test/functional/feature_dbcrash.py

```diff
@@ -274,7 +275,7 @@ def run_test(self):
         self.log.info(f"Restarted nodes: {self.restart_counts}; crashes on restart: {self.crashed_on_restart}")
 
         # If no nodes were restarted, we didn't test anything.
-        assert self.restart_counts != [0, 0, 0]
+        assert_not_equal(self.restart_counts, [0, 0, 0])
```

Comment:
`assert_equal` is helpful because when `a == 3` fails, it can be helpful to know what the value of `a` actually is without having to add debug code and rerun. If `a != 3` fails, though, you already know that `a == 3`, so this doesn't seem very helpful.

There are cases where you're comparing `a != b` and may not be sure which ones getting the wrong value where this could perhaps be helpful, but those seem pretty rare.

Concept -0 for me.